### ISSUE TITLE
Make auto-pilot walk slower or custom speed

### ISSUE THREAD

**Issue by askovi** [iam4x/pokemongo-webspoof#67]
I noticed the autopilot walking at 11-12km/h barely registers anything for the eggs. I got like 400m after walking like 3km. However, if I manually walk at 6-7 km/h by pressing arrows slower, it registers almost everything.

**Comment by MoOx** [comment]
Exact same comment. Maybe 11-12 is too high, should be more around 10 (which is already fast for a human on a long distance, I can tell you that from my last footing using a speedometer haha)

**Comment by MoOx** [comment]
Custom range speed will be awesome. Also we could add a "car" speed, between 50 & 110km/h :) (convenient in country land).

**Comment by wxrdn** [comment]
agree. would be awesome if we can just customize the speed. currently default speed for subway is too low.

**Comment by muescha** [comment]
let user add different types:
- slow walking
- bicycle
- fligth
etc

**Comment by iam4x** [comment]
Will make walking back to 10km/h which was working great with eggs 👍

**Comment by MoOx** [comment]
10.5 works great for me ;) Maybe we can try 11? (note: with 10.5, sometimes I see 10.6... that might be the issue). Maybe 11.5 ? :D

**Comment by wxrdn** [comment]
Please give us CUSTOME SPEED, like 0 to 200 km/h, that would be so sweet! The moving speed is too low right now, 30 ~ 40 km on highway, that's a bit too conservative isn't it?

**Comment by iam4x** [comment]
* [x] Multiple speed and lower walking speed done into #130
* [ ] Still missing custom speed

**Comment by iam4x** [comment]
Released multiple speed: https://github.com/iam4x/pokemongo-webspoof/releases/tag/v1.1.1
